satpy/composites/__init__.py

```diff
@@ -59,6 +60,8 @@
 MASKING_COMPOSITOR_METHODS = ['less', 'less_equal', 'equal', 'greater_equal',
                               'greater', 'not_equal', 'isnan', 'isfinite',
                               'isneginf', 'isposinf']
+NIR_REFLECTANCE_TERMINATOR_LIMIT = 85.0
+NIR_REFLECTANCE_MASKING_LIMIT = 88.0
```

Comment:
Just to keep things gathered in the same place, would it be worth have these at *class* attributes to `NIRReflectance`?

Reply:
Will do. Update follows in a minute or three.

Reply:
Code and tests updated, tested with real data.